orig/manifests/cdi/cdi-operator.yaml

```diff
@@ -31,7 +31,7 @@ roleRef:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: "cdi-operator-leader-election-helper"
+  name: "cdi-controler-leader-election"
```

Comment:
is `controler` a typo?

Reply:
Not sure, I copied this as is from the generated manifests out of the v1.5 branch.

It shouldn't really affect anything, so we should be fine merging it, don't we?